### PR TITLE
Feat: Home info screen: new text, image, and external link

### PR DESCRIPTION
**Pull request by knipec**
## Description
- Style the component for external links (LinkNewWindowIcon.tsx)
- Refactor to move icon components to their own files (this is why there are so many files)
- Update home info sheet to match Figma design (LandPKSInfo.tsx)
- (Also updated the empty sites list screen to remove the link... also updated content and noted an issue with BulletList layout, which was tangential to the Issues this primarily addresses, whoops.) 

### Checklist
- [x] Corresponding issue has been opened

### Related Issues
Implements #801, #954

### Verification steps
Confirm that the info sheet on the home page looks like the Figma design
Confirm that the styling of the external link(s) in the LandPKS Data Privacy screen looks correct
(Also, confirm that the empty sites list looks reasonable / like the Figma as of 2024-04-24)

### CODE REVIEW

File: dev-client/src/components/icons/Icon.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2023 Technology Matters
```
Comment (paulschreiber) [inline dev-client/src/components/icons/Icon.tsx:2]:
```suggestion
 * Copyright © 2024 Technology Matters
```

File: dev-client/src/components/icons/Icon.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    color?: ThemeColor | string;
+  };
+
+export type IconName = IconProps['name'];
+
+export const Icon = ({size = 'md', color, ...props}: IconProps) => {
+  return (
+    <MaterialIcon
+      {...convertNBStyles(props)}
+      size={typeof size === 'string' ? theme.components.Icon.sizes[size] : size}
```
Comment (paulschreiber) [inline dev-client/src/components/icons/Icon.tsx:43]: question: do we have to check if `sizes[size]` exists and have a fallback?
```suggestion
      size={typeof size === 'string' && Object.hasOwn(theme.components.Icon.sizes, size) ? theme.components.Icon.sizes[size] : size}
```

File: dev-client/src/screens/HomeScreen/components/LandPKSInfo.tsx [hunk 3]
```diff
@@ -56,11 +51,11 @@ export const LandPKSInfo = () => {
         <Column>
           {[1, 2, 3].map(index => (
             <HStack key={index}>
-              <Text variant="body1" mr={2}>
-                {index + 1}
+              <Text variant="body1" mr={numberedListMargin}>
```
Comment (paulschreiber) [inline dev-client/src/screens/HomeScreen/components/LandPKSInfo.tsx:54]: suggestion: can we use https://github.com/jsamr/react-native-li? This looks useful for bulleted lists, too.

File: dev-client/src/components/icons/Icon.tsx (hunk 2 above)
Comment (knipec) [reply dev-client/src/components/icons/Icon.tsx:43]: I don't think that's necessary; the type checking in IconProps is already doing that constraint with `size?: keyof typeof theme.components.Icon.sizes | number;`

**Review by paulschreiber — APPROVED** [review]

**Comment by knipec** [comment]
This should be verifiable in v119

**Pull request merged**